test/lit.cfg

```diff
@@ -1312,14 +1331,15 @@ if os.path.exists(static_libswiftCore_path):
 # default Swift tests to use the just-built libraries
 target_stdlib_path = platform_module_dir
 if not kIsWindows:
+        libdispatch_path = getattr(config, 'libdispatch_build_dir', '')
```

Comment:
Inconsistent indentation here (although really it's the other way around; @compnerd accidentally snuck some tabs into this file).

Reply:
Fixed.

Reply:
Actually, I can't just fix the added line:
```
  File "/home/yln/work/swift-source/swift/test/lit.cfg", line 1335
    if 'use_os_stdlib' not in lit_config.params:
    ^
IndentationError: unexpected indent
```
We can reformat the whole block/section, but I would like to do this in a separate commit.

Reply:
My point is that you should use tabs for now, sincee Saleem's code uses tabs.